website/src/content/index.mdx

```diff
@@ -58,7 +58,7 @@ Focus more on implementing the right features instead of configuration. Our open
 
 - [Application Kit](https://github.com/commercetools/merchant-center-application-kit)
 - [UI Kit](https://github.com/commercetools/ui-kit)
-- [commercetools Platform](https://docs.commercetools.com/api)
+- [Composable Commerce API](https://docs.commercetools.com/api)
```

Comment:
This is the only place where it's singular (at least amongst the changed files), so we could make this one plural to be consistent.

**As to when/why:**
We should definitely use the plural version in cases where the old 'platform' phrase would refer to the whole of commercetools including the Change History API, Machine Learning API, soon Checkout Anywhere, and so on.

I also prefer the plural in most cases when talking about the HTTP API alone. Even though it's technically a single API for now, the plural form helps reinforce the message that you're not required to use the whole thing. You could just be using the Carts API, the Products API, or the Discounts API. Probably less work for us later on when we split out Discounts.

That said, the plural/singular versions are currently quite inconsistent in the main docs repo. Platform API and Platform APIs were used across the docs to mean the same thing, and because of the volume of the changes we had to make in that repo, we decided to not focus too much on that now. 

So in short, **let's go with plural**, but if it's not perfect for everywhere that's also ok. What do you think @emmenko?

@adinakleine do you have anything to add?

Reply:
Thanks for your suggestion @melindafekete. I do agree. 💯

Reply:
Should we add this rule to our writing style rules?

Reply:
> Should we add this rule to our writing style rules?

@melindafekete Would be great if you could care for this. 🙇‍♀️

Reply:
Changed to plural. Marking this thread as resolved.